src/WorkoutsMain/WorkoutList/WorkoutList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './WorkoutList.css';
+
+export default function WorkoutList() {
+    return (
+        <div className="WorkoutList">
+            <h2 className="WorkoutListTitle">Exercise List</h2>
+            <main className="container">
+                <div className="container-content">Exercise list A</div>
+                <div className="container-content">Exercise list B</div>
+                <div className="container-content">Exercise list C</div>
+                <div className="container-content">Exercise list D</div>
+                <div className="container-content">Exercise list E</div>
+                <div className="container-content">Exercise list F</div>
+            </main>
+        </div>
+    )
```

Comment:
I would add a semicolon at the end of the line just to keep it consistent with App.js